apps/web/src/routes/Members/Members.js

```diff
@@ -120,6 +126,16 @@ function Members (props) {
               onClick: () => changeSort(k)
             }))}
           />
+          {canSeeJoinAnswers && (
+            <div className='flex items-center gap-2'>
+              <SwitchStyled
+                checked={showAnswers}
+                onChange={() => setShowAnswers(!showAnswers)}
+                backgroundColor={showAnswers ? '#0DC39F' : '#8B96A4'}
+              />
+              <span className='text-sm font-medium text-foreground/80'>{t('Show Answers')}</span>
```

Comment:
this should probably say something like Show Join Question Answers, or Show Join Answers

Reply:
Yeah, its just such a small space.... having that much text will make that button dominate